### PR TITLE
Feat : Design Client review section of mobile app development page

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [x] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI 
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**UI Video :**

https://github.com/canopas/canopas-website/assets/85920585/c6016e84-988a-4f81-82a9-6ff359505c2a

**Performance:**
**Desktop :**
Screenshot from 2023-07-05 14-21-18
**Mobile:**
Screenshot from 2023-07-05 14-22-53

### CODE REVIEW

**Review by cp-priyanka-g — DISMISSED** [review]

File: vue-frontend/src/components/mobile-app-development/ClientReview.vue [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<template>
+  <section
+    class="tw-pb-[115px] tw-pt-[100px] md:tw-pb-[150px] md:tw-pt-[200px]"
+  >
+    <div class="tw-container">
+      <p
+        class="v2-canopas-gradient-text tw-hidden tw-hidden tw-text-left tw-font-inter-medium tw-text-[1.5rem] tw-leading-[1.816rem] md:tw-block"
+      >
+        Testimonials
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/mobile-app-development/ClientReview.vue:9]: You give gradient class here but it only shows pink color.

**Review by cp-mansi-d — DISMISSED** [review]

File: vue-frontend/src/pages/mobile-app-development.vue [hunk 2]
```diff
@@ -41,13 +47,6 @@ export default {
   setup() {
     var seoData = config.MOBILE_APP_DEVELOPMENT_SEO_META_DATA;
     useMeta({
-      meta: [
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/mobile-app-development.vue:44]: This should be removes when page is ready to deploy on production not in develoment

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/mobile-app-development/ClientReview.vue [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      </div>
+      <div class="tw-hidden md:tw-flex md:tw-flex-row lg:tw-gap-[3rem]">
+        <swiper
+          :spaceBetween="30"
+          :loop="true"
+          :slidesPerView="1"
+          :autoplay="{
+            delay: 3000,
+          }"
+          class="mySwiper md:tw-w-[60%] lg:tw-w-[55%]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/mobile-app-development/ClientReview.vue:50]: Replace classname with relevant name

File: vue-frontend/src/components/mobile-app-development/ClientReview.vue [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    filteredReview() {
+      return this.reviews[this.currentSlide];
+    },
+  },
+
+  methods: {
+    startSlideTransition() {
+      setInterval(() => {
+        this.currentSlide = (this.currentSlide + 1) % this.reviews.length;
+      }, 3000);
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/mobile-app-development/ClientReview.vue:162]: Increase interval

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Pull request merged**